lib/stack_master/template_compiler.rb

```diff
@@ -1,8 +1,11 @@
 module StackMaster
   class TemplateCompiler
+    TemplateCompilationFailed = Class.new(RuntimeError)
 
     def self.compile(config, template_file_path)
       template_compiler_for_file(template_file_path, config).compile(template_file_path)
+    rescue StandardError
```

Comment:
@stevehodgkiss  I've added some code to display (as suggested in the last reply) the cause of an exception (if present), in Command.perform.